### ISSUE TITLE
Logo proposal

### ISSUE THREAD

**Issue by Batarian711** [ef-carbon/locale#18]
Hello. I design a logo for open source projects voluntarily. Do you want to help?

**Comment by mattyclarkson** [comment]
Sure thing, have at it. SVG would be ideal.

**Comment by Batarian711** [comment]
Stay on standby.  It will be nice.

**Comment by Batarian711** [comment]
Do you have an idea?

**Comment by Batarian711** [comment]
Represents the pen with the language. The circle represents people. I designed a minimalist logo. What do you think? Is there anything you want to add?

![locale](https://user-images.githubusercontent.com/41051393/42758252-95d4a99a-890b-11e8-9500-c39c9b8b01b4.jpg)

**Comment by mattyclarkson** [comment]
I like it. I usually think of logos to not contain text, what do you think? Could you rework the pen/person bit into a standalone logo?

**Comment by Batarian711** [comment]
I sent pr.

**Comment by mattyclarkson** [comment]
Thanks! Give me a few days, afk.

**Comment by Batarian711** [comment]
Is there a problem ?

**Comment by mattyclarkson** [comment]
Nah, just busy today and then away for the weekend 😄Won't be near a keyboard the PR!

**Comment by Batarian711** [comment]
okey thanks sir 😃
